providers/msGraph/.czrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We have created before a provider folder for MSTeams as `ms-teams`. Please follow the same pattern by renaming the folder of your provider to `ms-graph`. 🙏🏻